### PR TITLE
hide the custom logging view setting

### PR DESCRIPTION
**Pull request by devoncarew**
- hide the custom logging view setting

From analytics, 1.8% of people have this enabled.

@pq 

This will address https://github.com/flutter/flutter-intellij/issues/3952 (as well as the longer tail of issues - https://github.com/flutter/flutter-intellij/labels/logging).

### CODE REVIEW

**Comment by robertodoering** [comment]
I really liked the custom logging view. If it is only hidden, is there another way to enable it?

**Comment by jaumard** [comment]
I was searching a reason why the experimental logging view was gone but wasn't expecting this ^^
 
I find this logging view far more productive that the default one for big project. Would like to see it (or something similar) come back and replace the default one.

**Comment by devoncarew** [comment]
So:

- we weren't actively maintaining this view, and it had accumulated some https://github.com/flutter/flutter-intellij/issues?q=is%3Aissue+is%3Aopen+label%3Alogging over time
- there were not a lot of users - ~1.8% developers had enable the option to display the view
- we've enhanced the output of the regular console view, specifically to better display errors from the flutter framework and to display dart:developer `log()` messages

We have disabled the view for now.

> I find this logging view far more productive that the default one for big project.

Can you talk a bit about the main reasons you used it? I think that would help us either improve the regular console output more, or choose to re-invest in the alternative logging view.

**Comment by jaumard** [comment]
I think I already tell those points few time ago, probably in one of the accumulated issues :p lol 
But basically what's help me on a day to day basis with the experimental view is:

- Filtering by log level (I have an app that log a lot of stuff between http calls and SQL requests just on app start I already have hundreds or more logs) so filtering is a huge plus 
- Search box (when it was working lol) allowing me to narrow the filtering by logger name (or names with regex usage), again super useful when debugging a specific feature(s)
- Color syntax on logs level, to visually see what log is more important (errors, warnings...)

Basically this give the equivalent of the Logcat view for Android, which is super nice.

**Comment by fquintan** [comment]
I agree with @jaumard that the ability to filter logs either by log level or regex will be sorely missed.
The color syntax on logs level I believe can be achieved with some packages like [logger](https://pub.dev/packages/logger) though I haven't tried it myself
